apps/api/src/app/workflows-v2/usecases/preview/preview.usecase.ts

```diff
@@ -383,6 +389,26 @@ function sanitizeControlValuesByOutputSchema(
   return replaceInvalidControlValues(controlValues, errors);
 }
 
+function sanitizeControlValuesByLiquidCompilationFailure(
```

Comment:
we want to remove the invalid control value, here we replace it the same way we replace invalid control values that did not pass schema validation.